scripts/exglobal_stage_ic.sh

```diff
@@ -104,6 +104,15 @@ for MEMDIR in "${MEMDIR_ARRAY[@]}"; do
         ;;
     esac
 
+    # Ocean Perturbation Files
+    if [[ "${MEMDIR}" != "mem000" ]] && [[ "${OCN_ENS_PERTURB_FILES:-}" == "true" ]]; then
```

Comment:
This will fail for GFS, where MEMDIR is an empty string. Use this instead:
```suggestion
    # Extra zero on MEMDIR ensure we have a number even if the string is empty
    if (( 0${MEMDIR:3} > 0 )) && [[ "${OCN_ENS_PERTURB_FILES:-false}" == "true" ]]; then
```